src/page-form/components/TextInput.tsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
```

Comment:
This can be removed now! interesting that eslint didn't catch this 🤔

Reply:
Good catch!